e2e/harmony/new.e2e.ts

```diff
@@ -27,29 +27,32 @@ describe('new command', function () {
 
       helper.scopeHelper.reInitLocalScope();
       helper.scopeHelper.addRemoteScope(undefined, undefined, true);
-      helper.command.create('starter', 'workspace-example');
+      helper.command.create('standalone-starter', 'react');
       helper.bitJsonc.addToVariant('*', 'teambit.harmony/aspect', {});
       helper.command.install();
 
-      const starterPath = path.join(helper.scopes.remote, 'workspace-example/workspace-example.starter.ts');
+      const starterPath = path.join(helper.scopes.remote, 'react/react.starter.ts');
       const starterContent = helper.fs.readFile(starterPath);
-      const updatedIndex = starterContent.replace('teambit.react/templates/ui/text', `${helper.scopes.remote}/comp1`);
+      const updatedIndex = starterContent.replace(
+        'teambit.community/component-showcase',
+        `${helper.scopes.remote}/comp1`
+      );
       helper.fs.outputFile(starterPath, updatedIndex);
 
       helper.command.tagAllComponents();
       helper.command.export();
 
       helper.scopeHelper.cleanLocalScope(); // it deletes all content without bit-init
       helper.scopeHelper.addRemoteScope(undefined, undefined, true);
-      helper.command.new('template-example', `--aspect ${helper.scopes.remote}/workspace-example`);
+      helper.command.new('react-workspace', `--aspect ${helper.scopes.remote}/react`);
     });
     after(() => {
       helper.scopeHelper.removeRemoteScope(undefined, true);
     });
-    it('should generate a new workspace, import components and add them as new with no issues', () => {
+    it('should generate a new workspace, import components and add with no issues', () => {
       const wsPath = path.join(helper.scopes.localPath, 'my-workspace');
-      const status = helper.command.statusJson(wsPath);
-      expect(status.newComponents).to.have.lengthOf(1);
+      const list = JSON.parse(helper.command.runCmd(`bit list --json`, wsPath));
```

Comment:
can you change it back to check the status of the newComponents?

Reply:
helper.command.expectStatusToNotHaveIssues(wsPath) run bit status anyway and I am checking the same check on the bit list as you can see